palindrom-client.html

```diff
@@ -119,7 +119,13 @@
                     type: Number,
                     value: '60'
                 },
-
+                /**
+                * Interval in seconds until the page refreshes upon connection errors
+                */
+                pageReloadDelay: {
```

Comment:
To be consistent with the name of the property `pingIntervalS`, this property should be called `pageReloadDelayS`.

Since other properties don't have `page` in their name, I think that calling the property `fatalErrorReloadAfterS` would be the most appropriate.